tests/requirements.txt

```diff
@@ -1,4 +1,5 @@
 coveralls
+future
```

Comment:
this is not required anymore as far as I know, this comes from the py2 era

Reply:
On master it has been removed already so it will disappear during the merge